notebooks/summit/AuxTel/SITCOM-426/SITCOM-426_SQ_PSFandPhotometry_Validation.ipynb

```diff
@@ -0,0 +1,1190 @@
+       "border-left-style: solid;\n",
+       "font-family: var(--jp-code-font-family);\n",
+       "font-size: var(--jp-code-font-size);\n",
+       "line-height: var(--jp-code-line-height);\n",
+       "\"><span style=\"color: var(--jp-warn-color2)\">lsst.ts.utils.tai</span> <span style=\"color: var(--jp-info-color0)\">INFO</span>: Update leap second table</pre>"
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "data": {
+      "text/html": [
+       "<pre style=\"margin: 0.1em;\n",
+       "padding-left: 0.25em;\n",
+       "border-left-style: solid;\n",
+       "font-family: var(--jp-code-font-family);\n",
+       "font-size: var(--jp-code-font-size);\n",
+       "line-height: var(--jp-code-line-height);\n",
+       "\"><span style=\"color: var(--jp-warn-color2)\">lsst.ts.utils.tai</span> <span style=\"color: var(--jp-info-color0)\">INFO</span>: current_tai uses the system TAI clock</pre>"
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import yaml\n",
+    "import logging\n",
+    "import jsonschema\n",
+    "\n",
+    "from lsst.ts import salobj\n",
+    "from lsst.ts.observatory.control.auxtel import ATQueue"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "6914edc9-91e0-403a-8b5e-affccdd59a61",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "domain = salobj.Domain()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "8744dd35-e908-40f8-86f7-c27d1226068b",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<pre style=\"margin: 0.1em;\n",
+       "padding-left: 0.25em;\n",
+       "border-left-style: solid;\n",
+       "font-family: var(--jp-code-font-family);\n",
+       "font-size: var(--jp-code-font-size);\n",
+       "line-height: var(--jp-code-line-height);\n",
+       "\"><span style=\"color: var(--jp-warn-color2)\">ScriptQueue</span> <span style=\"color: var(--jp-info-color0)\">INFO</span>: Disabling authlist-based command authorization</pre>"
+      ],
+      "text/plain": [
+       "<IPython.core.display.HTML object>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "queue = ATQueue(domain)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "71f14a88-4e27-4fc6-a049-9e59762d320b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "script_description=\"Executing SITCOM-426 PSF & Photometry Validation\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "abd66b8a-ed7f-4f0d-ba02-e5211bf375f9",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "async def send_to_SQ(is_standard, script_path, script_config):\n",
```

Comment:
Is this really necessary? If you don't want to use `add_standard` and `add_external`, you can use `add` instead:

```
await queue.add(
    is_standard=take_cwfs_is_standard,
    ....
)
```

Reply:
Thanks! Great. I didn't see this `add` method when I was writing this :)